src/components/DistanceRequest.js

```diff
@@ -286,15 +282,34 @@ function DistanceRequest({transactionID, report, transaction, mapboxAccessToken,
                     />
                 )}
             </View>
-            <Button
-                success
-                style={[styles.w100, styles.mb4, styles.ph4, styles.flexShrink0]}
-                onPress={() => onSubmit(waypoints)}
-                isDisabled={_.size(validatedWaypoints) < 2 || (!isOffline && (hasRouteError || isLoadingRoute || isLoading))}
-                text={translate(isEditingRequest ? 'common.save' : 'common.next')}
-                isLoading={!isOffline && (isLoadingRoute || shouldFetchRoute || isLoading)}
-            />
-        </ScrollView>
+        </>
+    );
+
+    const content = (
+        <>
+            <View style={styles.flex1}>
+                <DraggableList
+                    data={waypointsList}
+                    keyExtractor={(item) => item}
+                    shouldUsePortal
```

Comment:
when should we use `shouldUsePortal={false}`

Is there any case when we don't have to use portal?

Reply:
ah that makes sense thank you for explanation